std_msgs/ColorRGBA raw
int16 proximity
string match
